config/overlays/odh/odh_model_controller_manager_patch.yaml

```diff
@@ -8,5 +8,8 @@ spec:
       containers:
         - args:
             - --leader-elect
-          image: $(odh-model-controller) 
+            - --model-registry-inference-reconcile
+            - --model-registries-namespace
+            - "odh-model-registries"
```

Comment:
I have a suspicious this might not be updated downstream (if compared here, to midstream) and to avoid headaches it would be better to read this from the DSC ?

Reply:
When do you think makes sense to read this information from DSC?

Midstream we can do that when we instantiate the controller but if anything changes on DSC we won't update this constant until we restart the pod

If we want to do that in the reconcile loop we have a problem because in upstream there's no concept of DSC

Reply:
I believe doing it at Controller init time to be sufficient imho.

Worst case eventually, default namespace related values could be refactored as a function, so that upstream just return the value, and midstream another function is passed which can do the lookup as needed. But again, I'm missing to identify a case atm where this would be needed. Happy to hear otherwise!

Reply:
https://github.com/opendatahub-io/odh-model-controller/pull/326/commits/fafd6021f103138cf0779f7585ecf58509583965 implements taking the namespace at controller init time, feels free to resolve this conversation if it looks ok to you